src/commonComponents/button/Button.js

```diff
@@ -0,0 +1,45 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Constants
+import { buttonVariants } from './constants/button.general';
+
+//Helpers
+import _noop from 'lodash/noop';
+import _keys from 'lodash/keys';
+import _lowerCase from 'lodash/lowerCase';
+import cx from 'classnames';
+
+//CSS
+import './button.css';
+
+function Button(props) {
+  const { label, onClick, className, variant, id } = props;
```

Comment:
provide default variant as: const {variant = buttonVariants.PRIMARY, .....} = props;

Reply:
we have default props right